README.md

```diff
@@ -174,7 +197,8 @@ val adminUsers: List<User> = db.findAll(
 ```
 
 ## executeUpdate
-```kotlin
```

Comment:
why the removal of the kotlin language hint?

Reply:
Because it's just the signature and not a full method it comes up as compilation errors in IntelliJ.

Reply:
ah k 👍